instances/middleware/.consumer.env.example

```diff
@@ -5,3 +5,7 @@ VARIABLE_NAME=consumer
 LOG_LEVEL=INFO
 SUPPORT_BASE_URL=
 SUPPORT_TOKEN=
+
+UCAM_URI=""
```

Comment:
It is ever so slightly ahead of schedule (UCAM isn't pulled into pipeline master yet), but agreed.